lib/galaxy/datatypes/binary.py

```diff
@@ -1869,6 +1869,12 @@ class H5MLM(H5):
     max_peek_size = 1000  # 1 KB
     max_preview_size = 1000000  # 1 MB
 
+    _CONFIG = '-model_config-'
+    _HTTP_REPR = '-http_repr-'
+    _HYPERPARAMETER = '-model_hyperparameters-'
+    _REPR = '-repr-'
+    _URL = '-URL-'
+
```

Comment:
Very minor, but do we need these leading underscores? That makes it a "private constant", but why mark it as private if it's a constant? Or is there some other meaning implied here?

Reply:
Updated. Thanks!